ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -114,12 +181,156 @@ class LargeTitleView: UIView {
     private var smallMorphingAvatarViewAnimator: UIViewPropertyAnimator? // responsible for animating the transform of smallMorphingAvatarView
     private var smallAnimatorRunningObserver: NSKeyValueObservation? // observes the running property, done to accomplish a "completion" for a pausesOnCompletion = YES animator
 
-    private let titleButton = UIButton() // button used to display the title of the current navigation item
+    private let titleAreaView = UIView(frame: .zero)
+    private var titleAccessoryType: NavigationBar.TitleAccessoryType = .none
+    private var subtitleAccessoryType: NavigationBar.TitleAccessoryType = .none
+
+    private lazy var titleStackView: UIStackView = {
+        let stackView = UIStackView(frame: .zero)
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+
+        return stackView
+    }()
+
+    // button used to display the title of the current navigation item
+    private lazy var titleButton: UIButton = {
+        let titleButton = UIButton(frame: .zero)
+        titleButton.translatesAutoresizingMaskIntoConstraints = false
+        titleButton.setTitle(nil, for: .normal)
+        titleButton.titleLabel?.font = Constants.titleFont
+        titleButton.setTitleColor(colorForStyle, for: .normal)
+        titleButton.titleLabel?.textAlignment = .left
+        titleButton.contentHorizontalAlignment = .left
+        titleButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        titleButton.addTarget(self, action: #selector(LargeTitleView.titleButtonTapped(sender:)), for: .touchUpInside)
+        titleButton.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+
+        return titleButton
+    }()
+
+    // button used to display the subtitle of the current navigation item
+    private lazy var subtitleButton: UIButton = {
```

Comment:
Can't we leverage from the TwoLineTitleView instead of recreating this layout on the LargeTitleView?

We can follow up with design but the individual handlers for the title/subtitle can't be used simultaneously because of the minimum tappable area of 44ptx44pt (https://developer.apple.com/design/tips/).

Reply:
It's proving very difficult and expensive to switch to TwoLineTitleView.
TwoLineTitleView doesn't support avatar view and updating based on navigationItem properties. Also, we need to refactor a lot of NavigationBar logic to swap between LargeTitleView and TwoLineTitleView.
I'm going to close this PR as a result and, perhaps, at a later time, revisit the whole thing.